packages/valory/services/impact_evaluator/service.yaml

```diff
@@ -304,7 +310,7 @@ extra:
         twitter_api_bearer_token: ${TWITTER_API_BEARER_TOKEN:str:null}
         twitter_mentions_endpoint: ${TWITTER_MENTIONS_ENDPOINT:str:2/users/1450081635559428107/mentions?}
         twitter_mentions_args: ${TWITTER_MENTIONS_ARGS:str:tweet.fields=author_id&user.fields=name&expansions=author_id&max_results=100&since_id={since_id}}
-        twitter_max_pages: 10
+        twitter_max_pages: ${TWITTER_MAX_PAGES:int:1}
```

Comment:
I would suggest we keep this to 1 for the time being since setting this to anything greater than 1 would mess up the API call tracker